src/zui/ZUIEditTextInPlace/index.tsx

```diff
@@ -44,18 +51,28 @@ const useStyles = makeStyles((theme) => ({
 
 export interface ZUIEditTextinPlaceProps {
   allowEmpty?: boolean;
+  autoOnEdit?: boolean;
+  currentType?: string;
   disabled?: boolean;
   onChange: (newValue: string) => void;
   placeholder?: string;
   value: string;
+  showBorder?: boolean;
+  setShowBorder?: (value: boolean) => void;
```

Comment:
These look like (and are indeed used like) state from `useState()`, but components should not be designed to directly set other components state. Instead, the common practice is to communicate upwards using `onXYZ` callbacks, that communicate what _happened_, not what the parent component should do in effect. In this case, a name like `onFocus` might make sense, i.e. a callback that communicates to the surroundings that this component was focused and went into edit mode.